js/cli/src/matches.ts

```diff
@@ -69,7 +69,7 @@ programCommand("create_match")
       );
     }
 
-    await anchorProgram.createMatch(createMatchArgs, {}, config.oracleState);
```

Comment:
No more oracleState?

Reply:
Its used in the args, but the last 2 args to createMatch weren't actually being used so I abstracted them away.

https://github.com/raindrops-protocol/raindrops/blob/81c0a8ef80a1cd7bd11393e3555d8da2253b8fb0/js/cli/src/matches.ts#L38-L49